lib/targets.go

```diff
@@ -17,10 +18,10 @@ import (
 
 // Target is an HTTP request blueprint.
 type Target struct {
-	Method string
-	URL    string
-	Body   []byte
-	Header http.Header
+	Method string      `json:"method"`
+	URL    string      `json:"url"`
+	Body   []byte      `json:"body"`
+	Header http.Header `json:"header"`
```

Comment:
Always uncertain which one is canonical `Header` or `Headers`.

Reply:
Usually I consult the rfc and it's consistently referring to that part of request/response as header, so all good here. :>